src/utils/dateUtils.ts

```diff
@@ -0,0 +1,9 @@
+import moment from 'moment-timezone';
+import 'moment/locale/es';
+
+const TIMEZONE = 'America/Argentina/Buenos_Aires';
+
+const argentinaMoment = (date?: string | number) => moment.tz(date, TIMEZONE);
+
+export const formatDate = (date: string, withTime?: boolean, separator?: string) =>
+  argentinaMoment(date).format(`DD · MM · YYYY${withTime ? ` [${separator}] HH:mm[hs]` : ''}`);
```

Comment:
argentinaMoment(date).format(`DD · MM · YYYY${withTime && ` [${separator}] HH:mm[hs]` }`);

Reply:
this is not possible, it generates conflicts
- ![Screen Shot 2021-07-15 at 11 52 47 PM](https://user-images.githubusercontent.com/40371687/125893727-d451da81-bf8a-4713-9e42-cc10675e2d94.png)